test/unit/utils/FilterUtilsSpec.js

```diff
@@ -22,6 +23,16 @@ describe('utils/FilterUtils', function() {
         };
         assert.strictEqual(FilterUtils.getCoercionType(filter), "String");
       });
+
+      it('String type when property_value can be parsed into a date', function () {
```

Comment:
Perhaps this test should be described as `'Boolean type when filter exists'` ? ... or something like that